src/view/MainViewManager.js

```diff
@@ -1683,8 +1683,13 @@ define(function (require, exports, module) {
 
         return result;
     }
-    
-    
+
+    function _findProvider(editor){
+        //true = checking for edit provider
+        //false = checking for doc provider
+        return (EditorManager.findProvider(editor,true)||EditorManager.findProvider(editor,false));
```

Comment:
Yeah, this is wrong.  We're going to want to iterate through the list of registered providers (wherever they are held) and see if any can provide what we need.  We can't assume there are only 2 to check.